backend/rpc.py

```
@@ -1,29 +1,47 @@
+  args_names = func.__code__.co_varnames[:func.__code__.co_argcount]
+  @wraps(func)
+  def wrapper_func(payload: str):
+    data = json.loads(payload, object_hook=json_reviver)
+    args = itemgetter(*args_names)(data)
+    ret = func(*args) if isinstance(args, tuple) else func(args)
+    if ret: return json.dumps(ret, default=json_replacer)
+  return wrapper_func
+
+
 class RPC:
   @staticmethod
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)